src/react-components/ui-root.js

```diff
@@ -281,12 +280,12 @@ class UIRoot extends Component {
 
       // We are not in mobile chrome, so launch into chrome via an Intent URL
       const location = window.location;
-      const qs = queryString.parse(location.search);
-      qs.vr_entry_type = "daydream"; // Auto-choose 'daydream' after landing in chrome
+      const qs = new URLSearchParams(location.search);
+      qs.set("vr_entry_type", "daydream"); // Auto-choose 'daydream' after landing in chrome
 
       const intentUrl =
         `intent://${location.host}${location.pathname || ""}?` +
-        `${queryString.stringify(qs)}#Intent;scheme=${(location.protocol || "http:").replace(":", "")};` +
+        `${qs.toString()}#Intent;scheme=${(location.protocol || "http:").replace(":", "")};` +
```

Comment:
you can remove the `.toString()` - the template string literal will evaluate `qs` as a string

Reply:
since `location.protocol` will always be `https:` or `http:`, you could do `location.protocol.slice(0, -1)`; admittedly, that's not as expressive as the `.replace(…)` call. at the very least, you could remove the ` || "http:"` bit.

Reply:
> you can remove the .toString() - the template string literal will evaluate qs as a string

Oh, I didn't know that, thanks!

> since `location.protocol` will always be `https:` or `http:`, you could do `location.protocol.slice(0, -1)`; admittedly, that's not as expressive as the `.replace(…)` call. at the very least, you could remove the ` || "http:"` bit.

Hmm, I wonder what the person writing this was thinking -- how could `location.protocol` ever be falsy? I will double-check and probably remove it if I can't find any reason. As for the replacement, I think it's a little clearer the way it is.